tools/clang/test/DXC/dxa_tests.test

```diff
@@ -0,0 +1,36 @@
+
```

Comment:
I've been made to understand that `.test` shouldn't be used as a suffix for an hlsl file.  At any rate, it is inconsistently used among these files.

Reply:
This is not a hlsl file. The hlsl is in %S/Inputs/smoke.hlsl. This file just contains commands and FileCheck tags.